tests/container_disk_test.go

```diff
@@ -175,13 +175,29 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting from custom image location", func() {
 		Context("with disk at /custom-disk/downloaded", func() {
-			It("[test_id:1466]should boot normally", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirrosCustomLocation), "#!/bin/bash\necho 'hello'\n")
-				for ind, volume := range vmi.Spec.Volumes {
-					if volume.ContainerDisk != nil {
-						vmi.Spec.Volumes[ind].ContainerDisk.Path = "/custom-disk/downloaded"
+
+			withVolumePath := func(volName, path string) libvmi.Option {
```

Comment:
This function is a `libvmi.Option`, that only used here. I don't want to make it package function, to prevent package scope pollution.

Reply:
IMHO this is an abuse of lamda. No harm would happen if other member of this package can use a well-named function that has no side effects.

Reply:
I made it even more local and custom. We don't need this functionality anywhere else, and since we know the details, we don't even need the parameters, or the wrapper function. Also, using `Option` as the variable type, makes it, IMO, less "lambda" and more libvmi Option:
```go
const (
	volName = "disk0"
	path    = "/custom-disk/downloaded"
)

var withVolumePath libvmi.Option = func(vmi *v1.VirtualMachineInstance) {
	for _, vol := range vmi.Spec.Volumes {
		if vol.Name == volName && vol.ContainerDisk != nil {
			vol.ContainerDisk.Path = path
			break
		}
	}
}
```